docs/rst_sample.rst

```diff
@@ -0,0 +1,31 @@
+IAI Project Template
+======================
```

Comment:
Should be the same length as the line above

Reply:
Done.